packages/combo-box/test/filtering.test.js

```diff
@@ -300,18 +327,7 @@ describe('filtering items', () => {
       setInputValue('bar');
 
       expect(comboBox.opened).to.be.true;
-      expect(comboBox._focusedIndex).to.equal(1);
-    });
-
-    it('should reset focus when opening with filter cleared', () => {
-      comboBox.filteredItems = ['foo', 'bar', 'baz'];
-      comboBox.value = 'bar';
-      comboBox.close();
-
-      setInputValue('');
-
-      expect(comboBox.opened).to.be.true;
-      expect(comboBox._focusedIndex).to.equal(-1);
```

Comment:
**note:** This test has been dropped in favor of:

https://github.com/vaadin/web-components/pull/3600/files#diff-7fa09446be562a2e3920fce61ff91e9083fe0e11ea590a43a5e9dc9107f4da6dR412-R416